build.gradle

```diff
@@ -74,7 +75,7 @@ tasks.withType(ValidatePlugins).configureEach {
     enableStricterValidation = true
 }
 
-test {
+tasks.withType(Test).configureEach {
```

Comment:
I feel like I've heard different things about if the `.configureEach { }` is necessary.

Reply:
And I think it depends on Kotlin vs. Groovy DSL?

Reply:
I've also been confused by this.
I tested `tasks.withType(Test)` and it was eagerly created: https://ge.solutions-team.gradle.com/s/jdfll5ue7iv3i/performance/configuration?details=code-unit-build.gradle!:&openScriptsAndPlugins=WyJjb2RlLXVuaXQtYnVpbGQuZ3JhZGxlIl0